src/mono/wasm/runtime-test.js

```diff
@@ -62,12 +62,13 @@ if (typeof console !== "undefined") {
 		console.error = console.log;
 }
 
-if (typeof crypto == 'undefined') {
+if (typeof crypto === 'undefined') {
```

Comment:
I don't think this is a functional change since typeof should always return a string. Should we be using `===` throughout our JS? I've seen a mix checked in, but if we want to default to the versions without type conversion I'll keep an eye out for that.

Reply:
We should clean up these checks in general the match the js version we require but this is fine.